site/soc/software/config/versions.yaml

```diff
@@ -232,8 +266,21 @@ data:
         ks_endpoints: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_osh_image_version }}"
       divingbell: {}
       drydock: {}
-      ingress: {}
-      keystone: {}
+      ingress:
+        ingress_module_init: "{{ suse_osh_registry_location }}/openstackhelm/neutron:{{ suse_openstack_image_version }}"
+        ingress_routed_vip: "{{ suse_osh_registry_location }}/openstackhelm/neutron:{{ suse_openstack_image_version }}"
+      keystone:
+        bootstrap: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+        db_init: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+        keystone_db_sync: "{{ suse_osh_registry_location }}/openstackhelm/keystone:{{ suse_openstack_image_version }}"
+        db_drop: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
```

Comment:
kind of crazy to have this duplicated data here. Isnt there a better way without having to duplicate all those entries? Like have a generic images common to all included everywhere and then add extra images needed by each component?

Reply:
sounds like a good enhancement task even for upstream, but if should not happen, in socok8s. I would say a different task from this though.

Reply:
with that being said, we already use common vars for the registry and os version, this is pretty much fixed  and don't expect frequent changes in the list here.